src/components/ccsds_command_depacketizer/gen/generators/ccsds_commands.py

```diff
@@ -22,3 +22,51 @@ def generate(self, input_filename):
             a.additional_types.append("Ccsds_Command_Secondary_Header.T")
         assembly_gen.create_type_field_strings(a)
         print(a.render(self.template, template_path=self.template_dir))
+
+
+class assembly_ccsds_cosmos_commands_txt(assembly_plugin_generator, generator_base):
+    def __init__(self):
+        this_file_dir = os.path.dirname(os.path.realpath(__file__))
+        template_dir = os.path.join(this_file_dir, ".." + os.sep + "templates")
+        assembly_plugin_generator.__init__(
+            self, "name_ccsds_cosmos_commands.txt", template_dir=template_dir, subdir="plugin"
+        )
+
+    def generate(self, input_filename):
+        a = self.model_cls(input_filename)
+        if "Ccsds_Command_Depacketizer" in a.component_types_dict:
+            a.additional_types.append("Ccsds_Command_Secondary_Header.T")
+        assembly_gen.create_type_field_strings(a)
+        print(a.render(self.template, template_path=self.template_dir))
+
+
+class assembly_ccsds_cosmos_telemetry_txt(assembly_plugin_generator, generator_base):
```

Comment:
This generator should live with the Ccsds_Packetizer component, not this component, which is responsible for decoding CCSDS commands.

Reply:
Addressed in [f972337](https://github.com/lasp/adamant/pull/21/commits/f9723374b4b17059e6086b7cc2b590b1c5b5d824).